pydantic/validators.py

```diff
@@ -214,6 +215,15 @@ def path_exists_validator(v) -> Path:
     return v
 
 
+def callable_validator(v) -> Callable:
+    # Perform a simple check if the passed value is callable.
```

Comment:
```suggestion
    """
    Perform a simple check if the value is callable.

    Note: complete matching of argument type hints and return types is not performed
    """
```

(I can't do a multiline suggestion, so you'll have to manually remove the line below.)